android-core/src/main/java/com/mparticle/MParticleOptions.java

```diff
@@ -54,6 +54,7 @@ public class MParticleOptions {
     private MParticle.OperatingSystem mOperatingSystem = MParticle.OperatingSystem.ANDROID;
     private DataplanOptions mDataplanOptions;
     private Map<Class, List<Configuration>> mConfigurations = new HashMap();
+    private Map<Integer, Object> localKits = new HashMap<>();
```

Comment:
`localKits` should be called `sideloadedKits` to match the other platorms.

Also, why is this a map instead of an array like on the other platforms? Is there a specific need on Android for this?

Reply:
Changed